partner/partner-management-service/src/main/java/io/mosip/pms/oidc/client/dto/ClientDetailCreateRequestV2.java

```diff
@@ -0,0 +1,33 @@
+package io.mosip.pms.oidc.client.dto;
+
+import java.util.List;
+import java.util.Map;
+
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.NotEmpty;
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Size;
+
+import org.hibernate.validator.constraints.URL;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+public class ClientDetailCreateRequestV2 extends ClientDetailCreateRequest{
+
+	 @NotEmpty
+	 private Map<@Size(min=3, max=3) String, @NotBlank String> clientNameLangMap;
```

Comment:
Please handle varying size, if possible make it configurable.

Reply:
we want to limit the key size to 3 as the key will be lang code which is always a 3 letter code.